source/code/include/playfab/PlayFabIOSHttpPlugin.h

```diff
@@ -43,10 +43,10 @@ namespace PlayFab
 
             bool Initialize(std::unique_ptr<CallRequestContainerBase>& requestContainer);
             
-            enum State:int
+            enum class State:int
             {
                 None = 0,
-                Pending = RequestTask::None,
+                Pending = (int)RequestTask::State::None,
```

Comment:
>Pending [](start = 16, length = 7)

is this an existing bug, none and pending were both 0?

Reply:
It's not currently logged as a bug, should we consider it one?